src/packages/src/xroad/common/base/etc/xroad/services/global.conf

```diff
@@ -11,4 +14,18 @@ if [ -f /var/run/xroad/xroad-akka-env.properties ]; then
 fi
 
 export LD_LIBRARY_PATH="/usr/share/xroad/lib:$LD_LIBRARY_PATH"
-XROAD_PARAMS=" -XX:+UseG1GC -Dfile.encoding=UTF-8 -Xshare:auto -Djdk.tls.ephemeralDHKeySize=2048 "
+
+load_from_properties () {
+  if [ -f /etc/xroad/services/local.properties ]; then
+    while IFS='=' read -r key value
+    do
+      if [ -n "$key" ] && [ -n "$value" ] && [[ $key != \#* ]]; then
```

Comment:
As an additional precaution, it would be good to ignore "PATH"

Reply:
As discussed on slack lets go one step further and fix the list of environment variables that can be changed as this is the safest option.